src/ejercicio_10/uso/Main.java

```diff
@@ -0,0 +1,80 @@
+package ejercicio_10.uso;
+
+
+import imple.DiccionarioSimple;
```

Comment:
Si hacemos los import de la siguiente forma, optimizamos la cantidad:

```suggestion
import tda.*;
import imple.*;
```